bin/run_with_elasticache_redis3.sh

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env sh
-# due to having two routemasters in one hopper app we need a way to
-# set the redis url differently for some of them. this is how we do it.
+# allows setting the redis url per-service
 set -e
 
 export ROUTEMASTER_REDIS_URL=$ELASTICACHE3_ROUTEMASTER_REDIS_URL
```

Comment:
What do you think about changing app code to accept the Elasticache variable when provided? We can simply get rid of this file and simplify the Hopper config.

Reply:
The idea of this is you could set an app ENV VAR per service which i'm still not sure how you'd (simply) do in a generic enough way _inside_ the app itself.

Reply:
hmm. how would you pass `ELASTICACHE3_ROUTEMASTER_REDIS_URL` for each service with a different value?

Reply:
>hmm. how would you pass ELASTICACHE3_ROUTEMASTER_REDIS_URL for each service with a different value?

terraform would create additional env vars (perhaps the next one would be ELASTICACHE4_ROUTEMASTER_REDIS_URL). A service which mapped to that would use the appropriate override script (copy thereof).

Reply:
I think it would cleaner to do the following:

- update the app-infrastructure entry that sets the environment variable. have it set ROUTEMASTER_REDIS_URL in addition to ELASTICACHE3_ROUTEMASTER_REDIS_URL
- update the routemaster code to remove all logic we've added that set ROUTEMASTER_REDIS_URL to use value of ELASTICACHE3_ROUTEMASTER_REDIS_URL
- then update app-infrastructure to stop setting ELASTICACHE3_ROUTEMASTER_REDIS_URL

That, IMO, will give us the cleanest result. I don't _think_ we'll need to change Redis for Routemaster anymore. But if we do, we know exactly what is required.

Reply:
Ok, I understand what you mean now, and I agree that is very clean.
It gets us back to exactly the same state as before the migration.

Reply:
Part 1 rolled out.

Reply:
Part 2 rolled out.